app/containers/HomePage/index.js

```diff
@@ -128,11 +132,4 @@ const withConnect = connect(
   mapDispatchToProps,
 );
 
-const withReducer = injectReducer({ key: 'home', reducer });
-const withSaga = injectSaga({ key: 'home', saga });
-
-export default compose(
-  withReducer,
-  withSaga,
-  withConnect,
-)(HomePage);
+export default compose(withConnect)(HomePage);
```

Comment:
Do we want to use `React.memo()` here to account for the loss of extending from `PureComponent`? I know it's premature performance optimization, but was thinking about consistency.

Reply:
I'm fairly sure it provides no benefit at all in this particular case (neither did `PureComponent`) but I'll add it anyway because IMO it's not a commonly used API and we can promote its existence by using it in the sample app.

Remove two HOCs, add another one! 😅